packages/launch/_assets/layout.css

```diff
@@ -333,18 +346,6 @@ rocket-navigation > ul {
   position: relative;
 }
 
-#sidebar-nav li.anchor a:hover::before {
```

Comment:
Removed this for now because we don't really need both the vertical stripe, the hover color effect, and the octicon before content imo. Could add it later if it makes sense but to me it looked "too much"